.github/workflows/test.yml

```diff
@@ -2,6 +2,11 @@ name: BVT
 
 on:
   workflow_dispatch:
+    inputs:
+      ref:
```

Comment:
Do you really need this ref? Shouldn't it work automatically?

Reply:
What I am trying to do is target a specific version to repro any failures. Most recently, version MsQuic 2.4.3 seems to be failing the Kernel mode test `Tests::RunKernelMode: Basic/WithFamilyArgs.ClientBlockedSourcePort/1 failed `. 

What I am also considering is perhaps adding options to target specific tests, on specific platforms, and running it N times.